FueledUtils/LoadingState.swift

```diff
@@ -38,7 +58,22 @@ public enum LoadingState<Error: Swift.Error> {
 }
 
 public extension Action {
-	public func loadingState() -> SignalProducer<LoadingState<Error>, NoError> {
+	///
+	/// **Deprecated**: Please use `getSafely(at:)` instead.
+	///
+	/// Refer to the documentation for `getSafely(at:)` for more info.
+	///
+	@available(*, deprecated, renamed: "loadingState")
+	// The unused parameter allows to bypass the compiler error "Invalid redeclaration of 'loadingState'",
+	// while retaining backward compatibility
+	public func loadingState(_ unused: Void = ()) -> SignalProducer<LoadingState<Error>, NoError> {
+		return self.loadingState
+	}
+
+	///
+	/// Returns the current loading state for a given action.
+	///
```

Comment:
@stephanecopin I think the documentation should be changed to something like that:
`A producer for Signals that will send the action’s current LoadingState.`

Reply:
@heymansmile I'm actually thinking, wouldn't this make more sense to return a `Signal`? For consistency with the rest of the `Action` properties, but also because it doesn't make sense to `start()` this - it's something that is always happening when the action is created. What do you think?
cc @leontiy @notbenoit

Reply:
@stephanecopin No, because since state is a continuous thing we should be able to get this any time we bind loadingState to a property. SignalProducer sent initial values at start in contrast with Signals. So for example when reusing cells with cellModels which contain actions and we want to display current state of the actions we need to bind cell's controls to the signal producers of loading states. Does it makes sense?

Reply:
I'll also be amending the doc for `.default` above so it reflects exactly when it's sent:
```
Represents the state of an action was never started, has received any values, has completed successfully or was interrupted
```

Reply:
@stephanecopin It would be a kinda breaking change because the initial implementation doesn't send any values if the Action has not been started yet. From my point of view it would be enough for us to replace method with computed property but return a SignalProducer as in the original implementation. A developer then can do everything needed with it depending on a task.

Reply:
Haha, all right I'll revert it back to what it was 😉